core/src/main/java/com/alibaba/nacos/core/cluster/ServerMemberManager.java

```diff
@@ -77,93 +78,93 @@
  */
 @Component(value = "serverMemberManager")
 public class ServerMemberManager implements ApplicationListener<WebServerInitializedEvent> {
-    
+
     private final NacosAsyncRestTemplate asyncRestTemplate = HttpClientBeanHolder
             .getNacosAsyncRestTemplate(Loggers.CORE);
-    
+
```

Comment:
Please fix code style, code style file in `/style`

Reply:
Sorry, I'm going

Reply:
@haoyann Please help me review again